en/docs/tutorials/scenarios/scenario11-graphql.md

```diff
@@ -1,4 +1,42 @@
 # Scenario 11 - GraphQL Support
 
-!!! Note
-    Content to be added.
\ No newline at end of file
+## User Story
+
+**Quantis is more focused on providing more capability to the developer community. They expect the developer community to build their own mobile applications and web apps to use their APIs. To make this process much easier, Quantis wants to expose GraphQL API to the public.**
+
+**_Time to Complete : 5 mins_**
+
+![GraphQL description]({{base_path}}/assets/img/tutorials/scenarios/graphql-schema-import.png)
+
+WSO2 API Manager supports creating GraphQL APIs using the GraphQL schema. Following steps can be used to create a sample API.
+
+1. Log in to publisher portal [https://localhost:9443/publisher](https://localhost:9443/publisher) using apiprovider@quantis.com and password _user123_
+2. Select **Create API → Import GraphQL SDL**
+
+    ![Select GraphQL]({{base_path}}/assets/img/tutorials/scenarios/select-graphql.png)
+
+3. Import the **train.graphql** in the **/resources** and create the API. use `http://backend-service:8080/train-operations/graphql` as the backend endpoint url
```

Comment:
```suggestion
3. Import the **train.graphql** in the **/resources** and create the API. Use `http://backend-service:8080/train-operations/graphql` as the backend endpoint URL.
```

Let's use periods everywhere at the end of all these numbered items. Please apply everywhere.

Reply:
Will add a link to point to the overview page